src/v/kafka/server/group.h

```diff
@@ -921,39 +918,12 @@ class group final : public ss::enable_lw_shared_from_this<group> {
     model::term_id _term;
     absl::node_hash_map<model::producer_id, model::producer_epoch>
       _fence_pid_epoch;
-    absl::node_hash_map<model::producer_id, tx_data> _tx_data;
+    chunked_hash_map<model::producer_identity, ongoing_transaction>
```

Comment:
same comment as in group_stm, curious if fence can be rolled into the state

also some are chunked_hash_map and others are node_hash_map, perhaps switch to chm everywhere if it works?

Reply:
done the switch, let's talk about fence state